docs/element-definitions.md

```diff
@@ -18,30 +18,23 @@ The Base API Element contains four properties: `element`, `meta`, `attributes`,
 
     The `element` property defines the name of element. It MUST be a string that references an element, which SHOULD be defined.
 
-- `meta` (enum)
+- `meta`
 
-    The `meta` property is a reserved object for Refract-specific values. When `meta` is an object, it MAY contain elements itself. The element definition SHOULD be used when interacting with `meta` and its properties and values.
+    The `meta` property is a reserved object for Refract-specific values that MAY contain elements itself. The element definition SHOULD be used when interacting with `meta` and its properties and values.
 
-    - Members
-        - (object)
-            - `id` - Unique Identifier, MUST be unique throughout the document
-            - `ref` (Element Pointer) - Pointer to referenced element or type
-            - `classes` (array[string]) - Array of classifications for given element
-            - `title` (string) - Human-readable title of element
-            - `description` (string) - Human-readable description of element
-            - `links` (array[Link Element]) - Meta links for a given element
-        - (array[Member Element])
+    - `id` ([String Element][]) - Unique Identifier, MUST be unique throughout the document
```

Comment:
In that case the sentence `The element definition SHOULD be used when interacting with `meta` and its properties and values.` is bit confusing because properties are always just strings and not elements so when interacting with values the element should be used but with properties not. Or that's what I understand.